templates/blogs/details.rs.html

```diff
@@ -19,17 +19,36 @@
     }
 </div>
 <div class="h-feed">
-    <h1><span class="p-name">@blog.title</span> <small>~@fqn</small></h1>
-    <p>@blog.summary</p>
-    <p>
-        @i18n!(ctx.1, "There's one author on this blog: ", "There are {0} authors on this blog: "; authors.len())
-        @for author in authors {
-            <a class="author p-author" href="@uri!(user::details: name = author.get_fqn(ctx.0))">@author.name(ctx.0)</a>
-        }
-    </p>
-    <p>
-        @i18n!(ctx.1, "There's one article on this blog", "There are {0} articles on this blog"; total_articles)
-    </p>
+    @if blog.banner_id.is_some() {
+        <div class="cover" style="background-image: url('@Html(blog.banner_url(ctx.0).unwrap_or_default())')"></div>
+        <img class="hidden u-photo" src="@blog.banner_url(ctx.0).unwrap_or_default()"/>
+    }
+    <div class="h-card">
+        <div class="user">
+            <div class="flex wrap">
+                <div class="avatar medium" style="background-image: url('@blog.icon_url(ctx.0)');" aria-label="@i18n!(ctx.1, "{}'s icon"; &blog.title)"></div>
+                <img class="hidden u-photo" src="@blog.icon_url(ctx.0)"/>
```

Comment:
isn't the Avatar also optional?

Reply:
icon_url returns either the blog avatar, or the default one (see https://github.com/Plume-org/Plume/pull/460/files#diff-a4a32b57517e9459ff561610609f8044R364 )

Reply:
duh, thanks!